.github/workflows/wheels.yml

```diff
@@ -78,6 +78,7 @@ jobs:
       CIBW_SKIP: "${{ matrix.cibw.skip || '' }}"
       CIBW_ARCHS_LINUX: "${{ matrix.cibw.arch || 'auto' }}"
       CIBW_ARCHS_MACOS: "${{ matrix.cibw.arch || 'auto' }}"
+      CIBW_MANYLINUX_IMAGE: "${{ matrix.cibw.image || '' }}"
```

Comment:
I'm not sure you can set this environment variable, I understand that you could set `CIBW_MANYLINUX_X86_64_IMAGE` for example though from https://cibuildwheel.readthedocs.io/en/stable/options/.

Have you confirmed this works?

Reply:
Good catch! It definitely does not.  I _think_ it used to a long time ago, but I've definitely made this exact mistake multiple times before.